static/canjs.js

```diff
@@ -154,13 +154,6 @@ function init() {
 			var parentContainer = currentMenu.parentElement;
 			parentContainer.insertBefore(fragment, currentMenu);
 
-			// Move the social-side-container to inside the new sidebar
-			var sidebarElement = document.querySelector('canjs-sidebar');
-			var socialContainer = currentMenu.querySelector('.social-side-container');
-			var bitoviContainer = currentMenu.querySelector('.by-bitovi-container');
-			sidebarElement.appendChild(socialContainer);
-			sidebarElement.appendChild(bitoviContainer);
-
```

Comment:
I moved the containers to always be outside of the sidebar so this isn’t necessary.